README.md

```diff
@@ -2,23 +2,83 @@
 
 # mobx-state-tree
 
-_Opinionated, transactional, MobX powered state container combining the best features of the immutable and mutable world for an optimal DX_
-
 [![npm version](https://badge.fury.io/js/mobx-state-tree.svg)](https://badge.fury.io/js/mobx-state-tree)
 [![CircleCI](https://circleci.com/gh/mobxjs/mobx-state-tree.svg?style=svg)](https://circleci.com/gh/mobxjs/mobx-state-tree)
 [![Coverage Status](https://coveralls.io/repos/github/mobxjs/mobx-state-tree/badge.svg?branch=master)](https://coveralls.io/github/mobxjs/mobx-state-tree?branch=master)
-[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/mobx-state-tree)
+[![Join the community on Github Discussions](https://img.shields.io/badge/GitHub-Discussions-lightgrey)](https://github.com/mobxjs/mobx-state-tree/discussions)
```

Comment:
Switch back to Spectrum

Reply:
but why?

Reply:
@akaguny It was a core team decision, but we could revisit it if we had enough energy from the community behind switching. It was determined that Spectrum is working "well enough for now".